components/library/gc/Makefile

```diff
@@ -18,13 +18,13 @@ USE_DEFAULT_TEST_TRANSFORMS = yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= gc
-COMPONENT_VERSION= 8.2.6
+COMPONENT_VERSION= 8.2.8
 COMPONENT_REVISION= 1
 COMPONENT_SUMMARY= Boehm-Demers-Weiser garbage collector for C/C++
```

Comment:
Please delete COMPONENT_REVISION or set it 0 when you change COMPONENT_VERSION.

Reply:
Done.